### ISSUE TITLE
[TS] hmacSha256Hash base64Url encoding issue

### ISSUE THREAD

**Issue by ansf** [heroiclabs/nakama-common#88]
In Typescript

```Typescript
nk.base64UrlEncode(nk.hmacSha256Hash('testpayload', 'testsecret'), false)
```

generates
` 77-977-9XV5277-9Zm7vv71S77-9C--_ve-_vXMuIO-_vX3vv73vv73vv70WD--_vRjvv70QdCEDFg`.

Using Go directly

```Go
	mac := hmac.New(sha256.New, []byte("testsecret"))
	_, err := mac.Write([]byte("testpayload"))
	log.Print(base64.RawURLEncoding.EncodeToString(mac.Sum(nil)))
```

generate a different string (which seems to be the correct one):

`lJJdXnaVZm7GUuwL-KRzLiCRfbT7kBYPtxjCEHQhAxY`.

Do I use the API in a wrong way or is there some encoding issue going on?

My dependencies:
nakama-runtime: "github:heroiclabs/nakama-common#v1.23.0"
docker image: registry.heroiclabs.com/heroiclabs/nakama:3.12.0

**Comment by zyro** [comment]
@ansf A fix for this is already merged and will be part of the upcoming 3.13.0 release of Nakama. Keep an eye on [the releases page](https://github.com/heroiclabs/nakama/releases). 👍

**Comment by ansf** [comment]
Awesome! Any ETA yet?
